proxy/core/src/main/scala/io/cloudstate/proxy/AkkaActorRegisterFeature.scala

```diff
@@ -8,26 +8,25 @@ import scala.collection.JavaConverters._
 
 @AutomaticFeature
 final class AkkaActorRegisterFeature extends Feature {
-
+  val akkaActorClass = classOf[akka.actor.Actor]
```

Comment:
I got the impression that it's safer/better to use `access.findClassByName` rather than instantiating a class literal separately and then passing it.

What's the motivation behind this change (I can guess reasons, but good to know what they actually were in your case).

Reply:
@dwijnand I got that impression too, but then I looked at the substratevm implementations, and figured the value of having this show up in compilation is much better than having it pop up when/if something runs the actual native-image job. I.e. prioritizing fast feedback.

Reply:
In all seriousness: wouldn't it do both if we did `access.findClassByName(classOf[akka.actor.Actor].getName)`?  Build-time safety of the name, semantics safety of which class it fetches.

Reply:
@dwijnand That's fair enough, let me take a stab at that.